VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -461,6 +461,85 @@
                 "deprecated": false
             }
         },
+        "/suggestions/configuration/account/config": {
+            "get": {
+                "tags": [
+                    "SKU Approval Settings"
+                ],
+                "summary": "Get Account's Matcher Settings",
+                "description": "This endpoint only retrieves configuration information from matchers in the marketplace account. If the account has no matcher configured, the response will be `404 Not Found`. \n\nFor account approval configuration information, with response including specification information, see the [Get Account's Approval Settings](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions#get-/suggestions/configuration) endpoint.",
```

Comment:
Pensando na máxima de "every page is page one", que contexto você dá pros devs que chegaram de paraquedas nesse endpoint, pra entenderem tudo que precisam sobre ele? Senti falta de uma frase de início mais completa com mais contexto, do jeito que está é como se você já passasse batido por conceitos com o bonde andando. Perguntas que surgem, ao ler a primeira frase:
- O que é um matcher?
- Que configuration information? Configuração do que?
- Quem deve chamar esse endpoint, e qual o interesse de pegar infos sobre a configuração do matcher. 
- Qual a diferença entre account matcher settings vs. account approval configuration? 

Muitas dessas perguntas podem ser resolvidas linkando artigos do Help, por exemplo.